examples/pyomobook/gdp-ch/scont.py

```diff
@@ -0,0 +1,36 @@
+# scont.py
```

Comment:
I don't think you put the filename at the top of any other file. It's not a problem, but struck me as odd.

Reply:
It has to do with how the example is included in the book. Some of the examples are included as a series of code snippets and some as complete scripts so it can be convenient to point readers to the name of the example in the repository.